brightway/export_agb/export.py

```diff
@@ -274,6 +240,11 @@ def export_json(content, filename):
 
         processes = compute_lca(processes, lcas)
```

Comment:
If we don't use a `return` [there](https://github.com/MTES-MCT/ecobalyse-data/pull/8/files#diff-56ba324c25239c2075568e882529945821d726d688ecb17ab0bd188065bb69c8R186) and instead edit in place, then there's no need to store the result back in `processes` here.